### PR TITLE
Register snapshot for 9006c3c

### PR DESCRIPTION
**Pull request by flaper87**
r? @alexcrichton

### CODE REVIEW

File: src/snapshots.txt [hunk 1]
```diff
@@ -1,3 +1,12 @@
+S 2015-01-29 9006c3c
```
Comment (alexcrichton) [inline src/snapshots.txt]: I think this date is supposed to be 01-20 instead of 01-29

**Comment by alexcrichton** [comment]
One small nit and otherwise r=me

File: src/snapshots.txt (hunk 1 above)
Comment (FlaPer87) [inline src/snapshots.txt]: doh... why is the 9 right next to the 0?

**Pull request merged**